design/baremetal-operator/maintenance-mode.md

```diff
@@ -0,0 +1,87 @@
+<!--
+ This work is licensed under a Creative Commons Attribution 3.0
+ Unported License.
+
+ http://creativecommons.org/licenses/by/3.0/legalcode
+-->
+
+# Add support for Maintenance Mode flag
+
+## Status
+
+implementable
+
+## Summary
+
+Currently there is no way to prevent management of BaremetalHost resources
+after provisioning is completed.  The underlying Ironic API contains a
+maintenance-mode flag which could be used to disable management of
+BMH resources.
+
+## Motivation
+
+In a multi-tier deployment where one cluster deploys another, the "parent"
```

Comment:
I think in general - transfer ownership between clusters, right?

Reply:
There have been some discussions about moving hosts between clusters as part of auto-scaling, but there are some different implications when doing that (we would absolutely need to deprovision those hosts as part of the transfer, to avoid leaking data from one cluster to another, for example). So, I think for this proposal let's stay focused on the multi-tier deployment case.